vlt-lock.json

```diff
@@ -8,6 +8,7 @@
       "eslint": "^9.28.0",
       "graph-run": "^1.1.0",
       "lru-cache": "^11.1.0",
+      "minimatch": "^10.0.1",
```

Comment:
Noting that this seems to keep getting picked up based on the fact it's in the `pnpm` lockfile & running `pnpm install && vlt install` will reintroduce it (don't know if anyone else is running both these commands realistically the lockfiles should be pretty much 1:1). I'm leaving it in but can make a separate PR for it if we want to get nit picky :shrug: